resource/sizesuffix.go

```diff
@@ -0,0 +1,247 @@
+package resource
```

Comment:
How about putting it directly in forwarder. This is needed for config and CLI.

Reply:
ok